third_party/ros/roslaunch/BUILD.bazel

```diff
@@ -1,21 +1,42 @@
 load("@rules_python//python:defs.bzl", "py_binary", "py_library")
 load("@rules_ros_pip_deps//:requirements.bzl", "requirement")
 load("//build_tools:pylint.bzl", "pylint")
-load("//third_party:expand_template.bzl", "expand_template")
 
-expand_template(
+genrule(
     name = "deps",
-    out = "deps.py",
-    data = ["roscore.xml"],
-    substitutions = {"{roscore_xml_path}": "$(rootpath roscore.xml)"},
-    template = "deps.py.tpl",
+    outs = ["deps.py"],
+    cmd = """
+ROSMASTER=($(rootpaths @ros_comm//:rosmaster))
```

Comment:
Hacky solution IMO, it would be cleaner to go with a symlink as I originally proposed. Right now this is fine, but the contents of `$(rootpaths @ros_comm//:rosmaster)` could change and then this could break. To be fixed when needed.